.github/workflows/pathogen-repo-ci.yaml

```
@@ -226,7 +227,8 @@ jobs:
       #   approach. -trs ]
       #
       #   -trs, 28 April 2022
-      - uses: nextstrain/.github/actions/setup-nextstrain-cli@c1191de9d5e1a30e91d70b0fd1041d97ed1b2496
+      - name: Setup Nextstrain
```

_nit nit nit_

```suggestion
      - name: Set up Nextstrain
```

(Yes, I realize the action path is `setup-nextstrain-cli` not `set-up-nextstrain-cli` but what can I say but _mea culpa_…)

---

fixed.

---

but also, sans further commentary: 

```
> ag 'setup '
workflows/pathogen-repo-build.yaml.in
282:        name: Setup runtime ${{ inputs.runtime }}

workflows/pathogen-repo-build.yaml
308:      - name: Setup runtime ${{ inputs.runtime }}
385:      - name: Setup runtime ${{ inputs.runtime }}
443:      - name: Setup runtime ${{ inputs.runtime }}
502:      - name: Setup runtime ${{ inputs.runtime }}
561:      - name: Setup runtime ${{ inputs.runtime }}
643:      - name: Setup runtime ${{ inputs.runtime }}
```

---

Yeah, that's only one actual instance (in the `.in` file), but it's also wrong. (Not my fault though ;-)